codegenerator/cli/templates/static/codegen/src/InMemoryTable.res

```diff
@@ -141,7 +141,11 @@ module Entity = {
   }
 
   let setRow = set
-  let set = (inMemTable: t<'entity>, entityUpdate: Types.entityUpdate<'entity>) => {
+  let set = (
+    inMemTable: t<'entity>,
+    entityUpdate: Types.entityUpdate<'entity>,
+    ~shouldRollbackOnReorg,
```

Comment:
Passing this in here because using RegisterHandlers causes dependency cycle

Reply:
Nice, we had to get rid of `Config.getGenerated()` anyways